pyproject.toml

```diff
@@ -27,24 +26,42 @@ packages = [
     { include = "humanloop", from = "src"}
 ]
 
+
 [project.urls]
 Repository = 'https://github.com/humanloop/humanloop-python'
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "^3.9"
 httpx = ">=0.21.2"
 httpx-sse = "0.4.0"
 pydantic = ">= 1.9.2"
 pydantic-core = "^2.18.2"
 typing_extensions = ">= 4.0.0"
+parse = "^1.20.2"
+opentelemetry-sdk = "^1.27.0"
+opentelemetry-api = "^1.27.0"
+opentelemetry-instrumentation-openai = "^0.33.3"
+opentelemetry-instrumentation-cohere = "^0.33.3"
+opentelemetry-instrumentation-anthropic = "^0.33.3"
+opentelemetry-instrumentation-groq = "^0.33.3"
+opentelemetry-instrumentation-replicate = "^0.33.3"
 
-[tool.poetry.dev-dependencies]
+[tool.poetry.group.dev.dependencies]
+parse-type = "^0.6.4"
+anthropic = "^0.37.1"
+groq = "^0.11.0"
+cohere = "^5.11.2"
```

Comment:
surprised we need to add these provider packages to our SDK - could you explain why this is needed?

Reply:
They are required for the integration tests. I have added them as dev dependencies so they shouldn't burden the end user.

Reply:
Ok, nice. To confirm, they wont install when user installs our package?

Reply:
I've followed the steps indicated in the `How to test` section of PR description and can confirm.